tests/providers/amazon/aws/operators/test_sagemaker_processing.py

```diff
@@ -255,3 +257,31 @@ def test_operator_defer(self, mock_job_exists, mock_processing):
         with pytest.raises(TaskDeferred) as exc:
             sagemaker_operator.execute(context=None)
         assert isinstance(exc.value.trigger, SageMakerTrigger), "Trigger is not a SagemakerTrigger"
+
+    @mock.patch.object(SageMakerHook, "describe_processing_job")
+    @mock.patch.object(SageMakerHook, "count_processing_jobs_by_name", return_value=0)
+    @mock.patch.object(
+        SageMakerHook,
+        "create_processing_job",
+        return_value={"ProcessingJobArn": "test_arn", "ResponseMetadata": {"HTTPStatusCode": 200}},
+    )
+    @mock.patch("airflow.providers.amazon.aws.operators.sagemaker.SageMakerBaseOperator._check_if_job_exists")
+    def test_operator_openlineage_data(self, check_job_exists, mock_processing, _, mock_desc):
+        check_job_exists.return_value = False
+        mock_desc.return_value = {
+            "ProcessingInputs": [{"S3Input": {"S3Uri": "s3://input-bucket/input-path"}}],
+            "ProcessingOutputConfig": {
+                "Outputs": [{"S3Output": {"S3Uri": "s3://output-bucket/output-path"}}]
+            },
+        }
```

Comment:
for consistency, you could put this in the annotation, or put the other returns values in the test body, but mixing the setup styles makes it harder to read

Reply:
Done.